src/controllers/AmazonController.js

```diff
@@ -35,15 +35,10 @@ controller = {
   getAlbumArt: function() {
     var img = document.querySelector("#nowPlayingSection .albumImage");
     if (!img) return "";
-    return img.src
+	return img.src && img.src.replace('45_', '500_');
```

Comment:
Ok good to know.  At this point i can't remember which are custom because
they needed to be and which are just because I did them before i had a
BasicController.

On Tue, May 19, 2015 at 10:47 AM Eric Nemchik <notifications@github.com>
wrote:

> In src/controllers/AmazonController.js
> <https://github.com/msfeldstein/chrome-media-keys/pull/45#discussion_r30625978>
> :
>
> > @@ -35,15 +35,10 @@ controller = {
> >    getAlbumArt: function() {
> >      var img = document.querySelector("#nowPlayingSection .albumImage");
> >      if (!img) return "";
> > -    return img.src
> > +	return img.src && img.src.replace('45_', '500_');
>
> Good luck, the other extensions that do media keys have trouble with
> amazon because it doesnt like mouse events or something. Whatever magic you
> have worked here is the only media key extension i've seen work on amazon.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/msfeldstein/chrome-media-keys/pull/45/files#r30625978>
> .
>